src/devices/Amg88xx/MovingAverageModeBit.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Amg88xx
+{
+    /// <summary>
+    /// Defines the bit(s) of the moving average mode register (addr: 0x07)
+    /// </summary>
+    public enum MovingAverageModeBit : byte
```

Comment:
Does this one needs to be public?
If yes, then use proper naming convention

Reply:
doesn't seem like it should be to me

Reply:
Can be internal, though renamed to MovingAverageModeBits, too.
I use the InternalsVisibleToAttribute to expose it to the unit test assembly.